src/client/components/Pagination/Pagination.stories.js

```diff
@@ -12,15 +12,16 @@ export default {
 
 const template = (args) => {
   // eslint-disable-next-line
-  const [propProducts, setPropProducts] = useState(args.products);
+  const [propProducts, setPropProducts] = useState([]);
   // eslint-disable-next-line
   const [currentRange, setCurrentRange] = useState(
     args.products.slice(0, args.productsPerPage),
   );
   // eslint-disable-next-line
   React.useEffect(() => {
+    setPropProducts(args.products);
     setCurrentRange(propProducts.slice(0, args.productsPerPage));
-  }, [propProducts, args.productsPerPage]);
+  }, [propProducts, args.productsPerPage, args.products]);
```

Comment:
Here we need to change the setCuurentRange so it slices the products array according to the page index we are on. My solution doesn't look very good but it does the job. If you can change it in a better way it whould be nice. 
`  React.useEffect(() => {
    setPropProducts(args.products);
    setCurrentRange(propProducts.slice(currentPageIndex * args.productsPerPage, args.productsPerPage + (currentPageIndex * args.productsPerPage)));
  }, [propProducts, args.productsPerPage, args.products, currentPageIndex]);`

Reply:
For me by doing this it shows like this
![image](https://user-images.githubusercontent.com/81110602/156796509-1ee4a69a-3cb1-471d-8348-0383c38d260c.png)

Reply:
Okay .. Thank you Florentina.. I updated as you suggested